abs.yaml

```diff
@@ -0,0 +1,7 @@
+# upload_channels:
+#   - sfe1ed40
```

Comment:
Should this package be uploaded to the `defaults`? It looks like it's a package of a general purpose and can be useful for testing by community.

Reply:
I do agree with you, especially because it is a small package, just needed for tests which I did on a side, but following protocol it should go to SF. I will wait for others' opinions before switching to `defaults`. @M-Waszkiewicz-Anaconda @JeanChristopheMorinPerso

Reply:
It should go on SF.